seedfarmer/commands/_deployment_commands.py

```diff
@@ -262,7 +262,7 @@ def _exec_deploy(mdo: ModuleDeployObject) -> ModuleDeploymentResponse:
 
 
 def prime_target_accounts(
-    deployment_manifest: DeploymentManifest, update_seedkit: bool = False, update_project_policy: bool = False
```

Comment:
Modifying this will violate our security profile.  ONLY the toolchain role can assume the deployment role(s).  By passing in a profile, you effectively alter the security posture.